src/components/HomeContainer/Management/Idea.module.css

```diff
@@ -0,0 +1,55 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-block: 24px;
+}
+
```

Comment:
Please add:
```
.container:last-child {
  margin-bottom: 0;
}
```
to prevent extra blank space being added to the layout by the last list item